modules/chempy/models.py

```diff
@@ -54,114 +52,29 @@ def get_residues(self):
                 list.append((start,c))
         return list
 
-#------------------------------------------------------------------------------
-
     def get_coord_list(self):
         lst = []
         for a in self.atom:
             lst.append(a.coord)
         return lst
 
-#------------------------------------------------------------------------------
     def get_mass(self):
         sm = 0.0
         for a in self.atom:
             sm = sm + a.get_mass()
         return sm
 
-#------------------------------------------------------------------------------
-    def get_nuclear_charges(self):
```

Comment:
Why is this and a bunch of other methods removed?

Reply:
Hi @JarrettSJohnson!
I wanted to discuss this after I finish, but it's probably better to do it as early as possible. 
I couldn't find the use of these functions anywhere, so maybe it's a good idea to remove this code?
If you think this code might still be useful, then I'll restore it

Reply:
It's a public API, so if any users out there are using them, their scripts would break. If this is intended, we should have least have a deprecation period and some alternative path to obtain a similar result.

Reply:
Copy that. 
I'll restore everything that's working properly